transport/server.cc

```diff
@@ -1539,6 +1539,10 @@ make_result(int16_t stream, messages::result_message& msg, const tracing::trace_
         response->set_frame_flag(cql_frame_flags::warning);
         response->write_string_list(msg.warnings());
     }
+    if (!msg.custom_payload().has_value()) {
```

Comment:
Did you mean the inverted condition:
```
if (msg.custom_payload().has_value()) {
```

Also, it's shorter to rely on cast to bool than `has_value() `:
```
if (msg.custom_payload()) {
```

Reply:
Yes, I noticed that, but forgot to push the change